snapshot/lib/snapshot/runner.rb

```diff
@@ -186,6 +190,13 @@ def launch(language, locale, device_type, launch_arguments)
       return Collector.fetch_screenshots(raw_output, dir_name, device_type, launch_arguments.first)
     end
 
+    def open_simulator_for_device(device)
+      return unless ENV['FASTLANE_EXPLICIT_OPEN_SIMULATOR']
```

Comment:
I'd like to try this behind a feature switch for now, and then possibly make it the default after we hear good things.